R/redcapConnection.R

```diff
@@ -236,6 +236,22 @@ redcapConnection <- function(url = getOption('redcap_api_url'),
                   length.out = rtry)
   rtry_q <- retry_quietly
 
+  # CSV delimiter state for this connection -------------------------
+  getCsvDelimiterAPI <- function(csv_delimiter) {
+  	csv_delimiter_api <- switch(
+  		csv_delimiter,
+  		","  = "comma",
+  		"\t" = "tab",
```

Comment:
the tab should be word "tab" but not "\t" based on document v16.0.4

<img width="1586" height="144" alt="Image" src="https://github.com/user-attachments/assets/0189544a-8fd5-4bfc-9886-bf44270d61fc" />